requirements/base.in

```diff
@@ -1,5 +1,10 @@
-unstructured[local-inference]>=0.7.1
+unstructured[local-inference]>=0.8.1
 unstructured-api-tools>=0.6.0
+pydantic<2.0.2
+# Pinning click due to a unicode issue in black
+# can remove after black drops support for Python 3.6
+# ref: https://github.com/psf/black/issues/2964
+click==8.1.3
```

Comment:
This was previously pinned in test.txt. Without pinning here, there's a version conflict between click v8.1.3 and v8.1.4.